routes/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (express) {
   router.use('/user', require('./users')(express))
   router.use('/post', require('./posts')(express))
   router.use('/feed', require('./feed')(express))
+  router.use('/search', require('./search')(express))
```

Comment:
Now *that* is something I don't mind the idea of. Different issue though. Open one :wink: